src/estimagic/visualization/colors.py

```diff
@@ -1,4 +1,11 @@
-import seaborn as sns
```

Comment:
This whole module should go. Are we still using it anywhere?

Reply:
Moved to ``src/dashboard``.